index.html

```diff
@@ -621,10 +623,30 @@ <h4>issuance</h4>
 
       <section>
         <h3>Instead of...</h3>
-        <p class="issue">This section needs to be defined.</p>
+        <p>
+          This representation can be obtained by:
+        </p>
+        <ol>
+          <li>
+            Completing the mapping for <a href="#in-addition-to">In Addition To...</a>.
+          </li>
+          <li>
+            Removing any properties from the <code>application/credential+ld+json</code> that were mapped.
+          </li>
+          <li>
+            Setting the resulting object to be the <code>vc</code> member of the claim set.
```

Comment:
```suggestion
            Complete the mapping for <a href="#in-addition-to">In Addition To...</a>.
          </li>
          <li>
            Remove any properties from the <code>application/credential+ld+json</code> that were mapped.
          </li>
          <li>
            Set the resulting object to be the <code>vc</code> member of the claim set.
```

Reply:
Step 2 *might* be better reworded to the following, but I'm not certain this is the meaning that was intended.

> `Remove any properties that were mapped from the <code>application/credential+ld+json</code>.`

Reply:
I think you suggestion is consistent, but not sure which is better for the reader.